### DISCUSSION TITLE
Make skinnable lements with text left/right justified/grow not from the center.

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22456] [Ideas]
Add the ability to make things like the combo counter grow to/from the left to be able to position them at the edge of the screen without them growing out of bounds for high combos and also not floating awkwardly far away from the sreen edge for low combos. same for all other things that can grow this way, ur cps longest combo etc. The ability to disable the text similar to the new judgement counter display would be awsome for things like combo too.

**Comment by Joehuu** [comment]
I believe this is already possible when you set the origin to `TopLeft` by right-clicking the element (if you are on mobile, you can't right-click yet).
